test.mk

```diff
@@ -161,50 +161,255 @@ spec:
+          responses:
+            '204':
+              description: Delete
+            default:
+              description: Error
+        patch:
+          operationId: Patch
+          responses:
+            '200':
+              description: Patch
+            default:
+              description: Error
+        post:
+          operationId: Post
+          responses:
+            '200':
+              description: Post
+            default:
+              description: Error
+        put:
+          operationId: Put
+          responses:
+            '200':
+              description: Put
+            default:
+              description: Error
+    security:
+    - google_jwt: []
+    securityDefinitions:
+      google_jwt:
+        authorizationUrl: ""
+        flow: "implicit"
+        type: "oauth2"
+        x-google-issuer: "https://cloud.google.com/iap"
+        x-google-jwks_uri: "https://www.gstatic.com/iap/verify/public_key-jwk"
+        x-google-audiences: "{{JWT_AUDIENCE}}"
+    host: "{{HOSTNAME}}"
+    x-google-endpoints:
+    - name: "{{HOSTNAME}}"
+      target: "{{IP_ADDRESS}}"
+
+--- 
+$(call TEST_INGRESS)
+---
+apiVersion: ctl.isla.solutions/v1
+kind: CloudEndpoint
+metadata:
+  name: {{NAME}}
+spec:
+  project: {{PROJECT}}
+  targetIngress:
+    name: {{INGRESS_NAME}}
+    namespace: default
+    jwtServices:
+    - {{NAME}}
+  openAPISpecConfigMap: 
+    name: swagger
+    key: spec
+--- 
+$(call TEST_APP)
+endef 
+
+define CHANGED_CONFIGMAP
+apiVersion: v1
+kind: ConfigMap 
+metadata: 
+  name: swagger 
+data: 
+  spec: |
+    swagger: "2.0"
+    info:
+      description: "wildcard config for any HTTP service."
+      title: "Template: {{NAME}} Changed"
+      version: "1.0.0"
+    basePath: "/"
+    consumes:
+    - "application/json"
+    produces:
+    - "application/json"
+    schemes:
+    - "http"
+    - "https"
+    paths:
+      "/**":
+        get:
+          operationId: Get
+          responses:
+            '200':
+              description: Get
+            default:
+              description: Error
+        post:
+          operationId: Post
+          responses:
+            '200':
+              description: Post
+            default:
+              description: Error
+        put:
+          operationId: Put
+          responses:
+            '200':
+              description: Put
+            default:
+              description: Error
+    security:
+    - google_jwt: []
+    securityDefinitions:
+      google_jwt:
+        authorizationUrl: ""
+        flow: "implicit"
+        type: "oauth2"
+        x-google-issuer: "https://cloud.google.com/iap"
+        x-google-jwks_uri: "https://www.gstatic.com/iap/verify/public_key-jwk"
+        x-google-audiences: "{{JWT_AUDIENCE}}"
+    host: "{{HOSTNAME}}"
+    x-google-endpoints:
+    - name: "{{HOSTNAME}}"
+      target: "{{IP_ADDRESS}}"
+endef 
 
 export TEST_APP
 service%-deployment.yaml:
 	@echo "$${TEST_APP}" | \
-	  sed -e "s/{{NAME}}/service$*/g" \
```

Comment:
Can you double check the whitespace in the make file? a lot of these look off in my vscode editor.